ldclient/config.py

```diff
@@ -261,6 +261,9 @@ def __init__(
         :param omit_anonymous_contexts: Sets whether anonymous contexts should be omitted from index and identify events.
         :param payload_filter_key: The payload filter is used to selectively limited the flags and segments delivered in the data source payload.
         """
+        if sdk_key and not validate_sdk_key(sdk_key, log):
+            raise ValueError("SDK key contains invalid characters")
```

Comment:
Aside from the excellent point above, do we know for sure that an invalid SDK key caused a raised exception in python? Can't remember if you told me you confirmed that or not, so apologies if I make you repeat yourself.